setup

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+source environment.conf
+
+if [ ! -d $promethee_path ]; then
+	git clone $promethee_url -b $promethee_branch
+	git --git-dir=$promethee_path/.git checkout --detach $promethee_version
+fi
+
+make -C $promethee_path
+
+if [ ! -d $tiff_utils_path ]; then
+	git clone $tiff_utils_url
+	git --git-dir=$tiff_utils_path/.git checkout --detach $tiff_utils_version
+fi
+
+g++ -std=c++14 $utils_path/comparator.cpp -o $utils_path/comparator
+
+g++ -std=c++14 $tiff_utils_path/tifGenerator.cpp -o $tiff_utils_path/tifGenerator -ltiff
+./$tiff_utils_path/tifGenerator $test_size $test_name.tif -R
+
```

Comment:
@Matheus7OP it seems this tif-converter.py could be made simpler. For example, it can receive a path instead of working in its current directory. If you do so, we avoid the `cp` and the `rm`